Contributing.md

```diff
@@ -94,6 +105,31 @@ Options:
   --verbose-browser-console  print browser console to stdout
 ```
 
+## Running the benchmarks
+
+First you'll want to `npm install`. To run all the benchmarks, use `npm run benchmark`.
+
+Using options to `npm run benchmark`, you can slice and dice which benchmarks your want to run. Usage is as follows:
+
+```
+$ npm run benchmark -- --help
+
+Run the jsdom benchmark suite
+
+Options:
+  -s, --suites  suites that you want to run. ie: -s dom/construction/createElement,dom/foo
+  --bundle      generate the javascript bundle required to run benchmarks in a browser
+  -h, --help    show the help
+```
+
+You can run the same benchmarks in chrome, in this case an automatic comparison will be made with the builtin DOM of chrome. First you will need to run:
+
+```
+npm run benchmark-browser
+```
+
+This tool will generate the necessary browser compatible javascript. After this command completes you can open `benchmark/browser-runner.html` in chrome. If you change a benchmark or if you change jsdom you will need to run this command again. You can then use the console in chrome (press F12) to run the benchmarks by executing the `run()` function.
```

Comment:
s/javascript/JavaScript; s/chrome/Chrome

Maybe `benchmark-browser` should launch `chrome benchmark/browser-runner.html`? Meh, maybe too inflexible for people without chrome in their path/on Chromium/etc.